### ISSUE TITLE
Tactical view just via 'enable/disable pixel improvements' button

### ISSUE THREAD

**Issue by Improver2000** [yairm210/Unciv#2633]
Hi there, 

First things first: your app is gorgeous!!

Currently with respect to overview I'm just missing one button left to the mini map: enable/disable pixel improvements. I like those pixel improvements a lot. But the bother while navigating units near cities :-(. So I simple 'tactical view' without pixel improvements would make game play much easier. Another option would be to combine 'enable/disable pixel improvements AND tile set default'. There to can't miss any unit cause then you have maximum overview :-).

Best wishes and keep on doing things so nice!!!

Ray

**Comment by yairm210** [comment]
Did you notice the option in menu-options to disable pixel improvements?

**Comment by Improver2000** [comment]
> Did you notice the option in menu-options to disable pixel improvements?

Yes,  I did.  There I got the expression 'pixel improvements' from. Within the game it is in my opinion too cumbersome to go to the options everytime. Since the overview would improve navigating units,  I guess it's useful to have such an option :-).
